Userland/Libraries/LibWeb/XHR/FormData.h

```diff
@@ -41,6 +41,9 @@ class FormData : public Bindings::PlatformObject {
     WebIDL::ExceptionOr<void> set(String const& name, String const& value);
     WebIDL::ExceptionOr<void> set(String const& name, JS::NonnullGCPtr<FileAPI::Blob> const& blob_value, Optional<String> const& filename = {});
 
+    // NON-STANDARD: Used to give access to the internal entry list.
+    Vector<FormDataEntry> entry_list() { return m_entry_list; }
```

Comment:
No need to copy, or alter, the vector :)

This should return a const reference and be a const method. The comment is not needed, it's a plain getter for a spec field.

Reply:
Can I blame forgetting to explicitly return a reference here that I have been traveling in C# land for the past weeks 🤦🏻‍♂️